src/Form/SettingsForm.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\timesup\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Times'up settings form.
+ */
+class SettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'timesup_admin_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form = parent::buildForm($form, $form_state);
+    $resolvers = $this->config('timesup.settings')->get('resolvers');
+
+    // Submitted form values should be nested.
+    $form['#tree'] = TRUE;
+
+    $form['resolvers'] = [
+      '#type' => 'fieldset',
+      '#title' => $this->t('Resolvers'),
+      '#description' => $this->t('Please enable resolvers you actively use to avoid cache/database invalidation stress.'),
+    ];
+
+    $form['resolvers']['minutely'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable Minutely resolver.'),
+      '#default_value' => !empty($resolvers['minutely']),
```

Comment:
Is there something missing here?

Reply:
It's a checkbox so true/false will check/uncheck it :) if !empty works on null and 0 value :)